### PR TITLE
Configure VM with VMM

### PR DESCRIPTION
**Pull request by tbazant**
rendered PDF is attached
[task-configure-virtual-machine-manager_color_en.pdf](https://github.com/SUSE/doc-modular/files/7098798/task-configure-virtual-machine-manager_color_en.pdf)

### CODE REVIEW

File: xml/task-configure-virtual-machine-manager.xml [hunk 1]
```diff
@@ -0,0 +1,1233 @@
...
+   To add new hardware, click <guimenu>Add Hardware</guimenu> below the left
+   panel, then select the type of the hardware you want to add in the
+   <guimenu>Add New Virtual Hardware</guimenu> window. Modify its parameters
+   and confirm with <guimenu>Finish</guimenu>.
+  </para>
+ </section>
+ <section xml:id="vmm-view-overview">
+  <title>Overview</title>
+  <para>
+   <guimenu>Overview</guimenu> shows basic details about &vmguest; and the
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:141]:
```suggestion
   <guimenu>Overview</guimenu> shows basic details about &vmguest; and the
```
```suggestion
   The <guimenu>Overview</guimenu> shows basic details about the &vmguest; and the
```

File: xml/task-configure-virtual-machine-manager.xml [hunk 2]
```diff
@@ -0,0 +1,1233 @@
...
+    </imageobject>
+    <imageobject role="html">
+     <imagedata fileref="libvirt_vmm_overview.png" width="70%" format="PNG"/>
+    </imageobject>
+   </mediaobject>
+  </figure>
+  <para>
+   <guimenu>Name</guimenu>, <guimenu>Title</guimenu>, and
+   <guimenu>Description</guimenu> are editable and help you identify &vmguest;
+   in the <guimenu>Virtual Machine Manager</guimenu> list of machines.
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:158]:
```suggestion
   <guimenu>Name</guimenu>, <guimenu>Title</guimenu>, and
   <guimenu>Description</guimenu> are editable and help you identify &vmguest;
   in the <guimenu>Virtual Machine Manager</guimenu> list of machines.
```
```suggestion
   The <guimenu>Name</guimenu>, <guimenu>Title</guimenu>, and
   <guimenu>Description</guimenu> fields are editable, and will help you to identify
   the &vmguest; in the <guimenu>Virtual Machine Manager</guimenu> list of machines.
```

File: xml/task-configure-virtual-machine-manager.xml [hunk 3]
```diff
@@ -0,0 +1,1233 @@
...
+    <imageobject role="fo">
+     <imagedata fileref="libvirt_vmm_desc.png" width="40%" format="PNG"/>
+    </imageobject>
+    <imageobject role="html">
+     <imagedata fileref="libvirt_vmm_desc.png" width="40%" format="PNG"/>
+    </imageobject>
+   </mediaobject>
+  </figure>
+  <para>
+   <guimenu>UUID</guimenu> shows the universally unique identifier of the
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:172]:
```suggestion
   <guimenu>UUID</guimenu> shows the Universally Unique IDentifier of the
```
```suggestion
   <guimenu>UUID</guimenu> shows the universally unique identifier of the
```

File: xml/task-configure-virtual-machine-manager.xml [hunk 4]
```diff
@@ -0,0 +1,1233 @@
...
+   virtual machine, while <guimenu>Status</guimenu> shows its current
+   status&mdash;<guimenu>Running</guimenu>, <guimenu>Paused</guimenu>, or
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:174]:
```suggestion
   status&mdash;<guimenu>Running</guimenu>, <guimenu>Paused</guimenu>, or
```
```suggestion
   status: <guimenu>Running</guimenu>, <guimenu>Paused</guimenu>, or
```

File: xml/task-configure-virtual-machine-manager.xml [hunk 5]
```diff
@@ -0,0 +1,1233 @@
...
+   <guimenu>Shutoff</guimenu>.
+  </para>
+  <para>
+   The <guimenu>Hypervisor Details</guimenu> section shows the hypervisor type,
+   CPU architecture, used emulator, and chipset type. None of the hypervisor
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:179]:
```suggestion
   CPU architecture, used emulator, and chipset type. None of the hypervisor
```
```suggestion
   CPU architecture, the emulator in use (if any), and the chipset type. None of the hypervisor
```

File: xml/task-configure-virtual-machine-manager.xml [hunk 6]
```diff
@@ -0,0 +1,1233 @@
...
+  <tip>
+   <title>Enabling disabled charts</title>
+   <para>
+    Not all the charts in the <guimenu>Graph</guimenu> view are enabled by
+    default. To enable these charts, go to
+    <menuchoice><guimenu>File</guimenu><guimenu>View
+    Manager</guimenu></menuchoice>, then select
+    <menuchoice><guimenu>Edit</guimenu><guimenu>Preferences</guimenu>
+    <guimenu>Polling</guimenu></menuchoice>, and check the charts that you want
+    to see regularly updated.
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:209]:
```suggestion
    to see regularly updated.
```
```suggestion
    to see.
```

Comment (lproven) [reply xml/task-configure-virtual-machine-manager.xml:209]: As I understand it, they will all be updated anyway, so it's just the ones you want to see -- is that right?

File: xml/task-configure-virtual-machine-manager.xml [hunk 7]
```diff
@@ -0,0 +1,1233 @@
...
+     <imagedata fileref="libvirt_vmm_polling_charts.png" width="40%" format="PNG"/>
+    </imageobject>
+   </mediaobject>
+  </figure>
+ </section>
+ <section xml:id="vmm-configure-basics">
+  <title>Basic hardware</title>
+  <para>
+   This section describes the setup of the virtualized processor and memory.
+   These components are vital to a &vmguest;, therefore you cannot remove them.
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:228]:
```suggestion
   These components are vital to a &vmguest;, therefore you cannot remove them.
```
```suggestion
   These components are vital to a &vmguest;, so you cannot remove them.
```

File: xml/task-configure-virtual-machine-manager.xml [hunk 8]
```diff
@@ -0,0 +1,1233 @@
...
+  </para>
+  <section>
+   <title>Processor</title>
+   <para>
+    <guimenu>CPU</guimenu> includes detailed information about &vmguest;
+    processor configuration.
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:234]:
```suggestion
    <guimenu>CPU</guimenu> includes detailed information about &vmguest;
    processor configuration.
```
```suggestion
    <guimenu>CPU</guimenu> includes detailed information about the processor
    configuration of the &vmguest;
   .
```

File: xml/task-configure-virtual-machine-manager.xml [hunk 9]
```diff
@@ -0,0 +1,1233 @@
...
+    models available in the drop-down box.
+   </para>
+   <para>
+    The host CPU model generally provides a good trade-off between CPU features
+    and ability to migrate the &vmguest;. &libvirt; does not model every aspect
+    of each CPU, so the &vmguest; CPU will not match the &vmhost; CPU exactly.
+    But the ABI provided to the &vmguest; is reproducible and during migration
+    the complete CPU model definition is transferred to the destination
+    &vmhost;, ensuring the migrated &vmguest; will see the exact same CPU model
+    on the destination.
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:270]:
```suggestion
    But the ABI provided to the &vmguest; is reproducible and during migration
    the complete CPU model definition is transferred to the destination
    &vmhost;, ensuring the migrated &vmguest; will see the exact same CPU model
    on the destination.
```
```suggestion
    However, the ABI provided to the &vmguest; is reproducible, and during migration,
    the complete CPU model definition is transferred to the destination
    &vmhost;, ensuring that the migrated &vmguest; will see the exact same CPU model
    on the destination.
```

File: xml/task-configure-virtual-machine-manager.xml [hunk 10]
```diff
@@ -0,0 +1,1233 @@
...
+   </para>
+   <para>
+    The <literal>host-passthrough</literal> model provides the &vmguest; with a
+    CPU that is exactly the same as the &vmhost; CPU. This can be useful when
+    the &vmguest; workload requires CPU features not available in &libvirt;'s
+    simplified <literal>host-model</literal> CPU. The host-passthrough model is
+    also required in some cases, for example, when running &vmguest;s with more
+    than 4TB of memory. The <literal>host-passthrough</literal> model comes
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:278]:
```suggestion
    than 4TB of memory. The <literal>host-passthrough</literal> model comes
```
```suggestion
    than 4&nbsp;TB of memory. The <literal>host-passthrough</literal> model comes
```

File: xml/task-configure-virtual-machine-manager.xml [hunk 11]
```diff
@@ -0,0 +1,1233 @@
...
+   <para>
+    After you activate <guimenu>Manually set CPU topology</guimenu>, you can
+    specify a custom number of sockets, cores and threads for the CPU.
+   </para>
+  </section>
+  <section>
+   <title>Memory</title>
+   <para>
+    <guimenu>Memory</guimenu> contains information about the memory that is
+    available to &vmguest;.
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:297]:
```suggestion
    available to &vmguest;.
```
```suggestion
    available to the &vmguest;.
```

File: xml/task-configure-virtual-machine-manager.xml [hunk 12]
```diff
@@ -0,0 +1,1233 @@
...
+      <imagedata fileref="libvirt_vmm_memory.png" width="70%" format="PNG"/>
+     </imageobject>
+    </mediaobject>
+   </figure>
+   <variablelist>
+    <varlistentry>
+     <term><guimenu>Total host memory</guimenu></term>
+     <listitem>
+      <para>
+       Total amount of memory installed on &vmhost;.
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:315]:
```suggestion
       Total amount of memory installed on &vmhost;.
```
```suggestion
       Total amount of memory installed on the &vmhost;.
```

File: xml/task-configure-virtual-machine-manager.xml [hunk 13]
```diff
@@ -0,0 +1,1233 @@
...
+       memory. Any change to this value will take effect after the next
+       &vmguest; reboot.
+      </para>
+     </listitem>
+    </varlistentry>
+   </variablelist>
+   <important>
+    <title>Large memory &vmguest;s</title>
+    <para>
+     &vmguest;s with memory requirements of 4TB or more currently need to use
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:343]:
```suggestion
     &vmguest;s with memory requirements of 4TB or more currently need to use
```
```suggestion
     &vmguest;s with memory requirements of 4&nbsp;TB or more currently need to use
```

File: xml/task-configure-virtual-machine-manager.xml [hunk 14]
```diff
@@ -0,0 +1,1233 @@
...
+    </imageobject>
+   </mediaobject>
+  </figure>
+  <para>
+   In the <guimenu>Autostart</guimenu> section, you can specify whether the
+   virtual machine should automatically start during the &vmhost; boot phase.
+  </para>
+  <para>
+   In the <guimenu>Boot device order</guimenu>, activate the devices that will
+   be used for booting &vmguest;. You can change their order with the up and
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:372]:
```suggestion
   be used for booting &vmguest;. You can change their order with the up and
```
```suggestion
   be used for booting the &vmguest;. You can change their order with the up and
```

File: xml/task-configure-virtual-machine-manager.xml [hunk 15]
```diff
@@ -0,0 +1,1233 @@
...
+      <imageobject role="html">
+       <imagedata fileref="libvirt_vmm_network.png" width="70%" format="PNG"/>
+      </imageobject>
+     </mediaobject>
+    </figure>
+   </step>
+   <step>
+    <para>
+     From the <guimenu>Network source</guimenu> list, select the source for the
+     network connection. The list includes &vmhost;'s available physical
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:559]:
```suggestion
     network connection. The list includes &vmhost;'s available physical
```
```suggestion
     network connection. The list includes the &vmhost;'s available physical
```

File: xml/task-configure-virtual-machine-manager.xml [hunk 16]
```diff
@@ -0,0 +1,1233 @@
...
+     network interfaces, network bridges, or network bonds. You can also assign
+     the &vmguest; to an already defined virtual network.
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:561]:
```suggestion
     the &vmguest; to a virtual network that was already defined.
```

File: xml/task-configure-virtual-machine-manager.xml [hunk 17]
```diff
@@ -0,0 +1,1233 @@
...
+    unless it is explicitly released (by pressing <keycombo>
+    <keycap function="alt"/> <keycap function="control"/> </keycombo>). To
+    prevent the console from grabbing the key and to enable seamless pointer
+    movement between host and guest instead, follow the instructions in
+    <xref linkend="proc-libvirt-config-input"/> to add an <guimenu>EvTouch USB
+    Graphics Tablet</guimenu> to the &vmguest;.
+   </para>
+   <para>
+    Adding a tablet has the additional advantage of synchronizing the mouse
+    pointer movement between &vmhost; and &vmguest; when using a graphical
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:641]:
```suggestion
    pointer movement between the &vmhost; and the &vmguest; when using a graphical
```

File: xml/task-configure-virtual-machine-manager.xml [hunk 18]
```diff
@@ -0,0 +1,1233 @@
...
+     Click <guimenu>OK</guimenu> to finish. The new media can now be accessed
+     in the &vmguest;.
+    </para>
+   </step>
+  </procedure>
+ </section>
+ <section xml:id="sec-libvirt-config-pci">
+  <title>Assigning a host PCI device to a &vmguest;</title>
+  <para>
+   You can directly assign host-PCI devices to guests (PCI pass-through). When
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:1013]:
```suggestion
   You can directly assign host-PCI devices to guests (PCI pass-through). When
```
```suggestion
   You can directly assign PCI devices on the &vmhost; to guests (PCI pass-through). When
```

File: xml/task-configure-virtual-machine-manager.xml [hunk 19]
```diff
@@ -0,0 +1,1233 @@
...
+   the PCI device is assigned to one &vmguest;, it cannot be used on the host
+   or by another &vmguest; unless it is re-assigned. A prerequisite for this
+   feature is a &vmhost; configuration as described in FIXME
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:1016]: Is this **FIXME** still outstanding? :-)

File: xml/task-configure-virtual-machine-manager.xml [hunk 20]
```diff
@@ -0,0 +1,1233 @@
...
+   <!-- <xref linkend="ann-vt-io-require"/> -->
+   .
+  </para>
+  <section xml:id="sec-libvirt-config-pci-vmm">
+   <title>Adding a PCI device with &vmm;</title>
+   <para>
+    The following procedure describes how to assign a PCI device from the host
+    machine to a &vmguest; using &vmm;:
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:1024]: Not sure if you want the &vmhost; entity here or are avoiding it intentionally?

File: xml/task-configure-virtual-machine-manager.xml [hunk 21]
```diff
@@ -0,0 +1,1233 @@
...
+  <title>Summary</title>
+  <para>
+   Your virtual machine is properly configured to suit your needs and you can
+   monitor its performance statistics.
+  </para>
+ </section>
+ <section xml:id="troubleshooting-configure-virtual-machine-manager">
+  <title>Troubleshooting</title>
+  <variablelist>
+   <varlistentry>
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:1158]: The `varlist` means we can't prepend "Error message:" or the like onto the front. This is a bit of a snag, IMHO. Maybe reformatting as plain paragraphs, or using nested `itemizedlist`s...?

File: xml/task-configure-virtual-machine-manager.xml [hunk 22]
```diff
@@ -0,0 +1,1233 @@
...
+   <listitem>
+    <para>
+     See FIXME
+     <!-- <xref linkend="sec-libvirt-storage-vmm"/> -->
+     for more details on managing storage with &vmm;.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     For more information on disk cache modes, see FIXME
```
Comment (lproven) [inline xml/task-configure-virtual-machine-manager.xml:1226]: 3 more **FIXME** entries here, just in case you had overlooked them.

**Review by lproven — CHANGES_REQUESTED** [review]

File: xml/task-configure-virtual-machine-manager.xml (hunk 19 above)
Comment (tbazant) [reply xml/task-configure-virtual-machine-manager.xml:1017]: yep

File: xml/task-configure-virtual-machine-manager.xml (hunk 22 above)
Comment (tbazant) [reply xml/task-configure-virtual-machine-manager.xml:1227]: they're intentional

File: xml/task-configure-virtual-machine-manager.xml (hunk 21 above)
Comment (tbazant) [reply xml/task-configure-virtual-machine-manager.xml:1159]: we can prepend a string inside the &lt;term&gt; normally

File: xml/task-configure-virtual-machine-manager.xml [hunk 23]
```diff
@@ -0,0 +1,1234 @@
...
+   feature is a &vmhost; configuration as described in FIXME
+   <!-- <xref linkend="ann-vt-io-require"/> -->
+   .
+  </para>
+  <section xml:id="sec-libvirt-config-pci-vmm">
+   <title>Adding a PCI device with &vmm;</title>
+   <para>
+    The following procedure describes how to assign a PCI device from the host
+    machine to a &vmguest; using &vmm;:
+   </para>
```
Comment (tbazant) [inline xml/task-configure-virtual-machine-manager.xml:1026]:
```suggestion
    The following procedure describes how to assign a PCI device from a &vmhost; to a &vmguest; using &vmm;:
   </para>
```

File: xml/task-configure-virtual-machine-manager.xml (hunk 20 above)
Comment (tbazant) [reply xml/task-configure-virtual-machine-manager.xml:1025]: replaced

**Review by tbazant — COMMENTED** [review]
included Liam's suggestions

**Pull request merged**